### PR TITLE
created form

### PR DESCRIPTION
**Pull request by romandivonchuk**
- [DEMO LINK](https://romandivonchuk.github.io/layout_html-form/)
- [TEST REPORT LINK](https://romandivonchuk.github.io/layout_html-form/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,7 +11,145 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST"
+    onsubmit="onSubmit()">
```
Comment (VinnieJ-2k20) [inline src/index.html:15]: See checklist, item 3
https://github.com/mate-academy/layout_html-form/blob/master/checklist.md
```suggestion
    <form
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="POST"
      onsubmit="onSubmit()"
    >
```

File: src/index.html [hunk 2]
```diff
@@ -11,7 +11,145 @@
...
+            min="0"
+            max="100"
+          >
+        </div>
+        <div class="form-field">
+          <label for="birthday">Full date of birth:</label>
+          <input type="date" name="birthday"
+          id="birthday"
+          size="20"
+          >
```
Comment (VinnieJ-2k20) [inline src/index.html:54]: Same - checklist, item 3.
Fix all inputs on the page.

File: src/style.css [hunk 3]
```diff
@@ -1 +1,16 @@
 /* styles go here */
+fieldset {
+  margin-bottom: 20px;
+}
+
+fieldset:last-child {
+  margin-bottom: 0;
+}
```
Comment (VinnieJ-2k20) [inline src/style.css:8]: Don't style using tags.
Add extra classes where necessary.

**Review by VinnieJ-2k20 — CHANGES_REQUESTED** [review]
You need to fix some code style and css selectors.

**Review by solaryasha — APPROVED** [review]
Good job !

**Pull request closed without merging**